release-please-config.json

```diff
@@ -161,6 +161,17 @@
     "plugins/source/test": {
       "component": "plugins-source-test"
     },
+    "plugins/source/typeform": {
+      "component": "plugins-source-typeform",
+      "release-type": "simple",
+      "extra-files": [
+        "plugins/source/typeform/version.py"
```

Comment:
I think this file needs to exist with a comment like in https://github.com/googleapis/python-aiplatform/blob/68e84e9bf97cabceb7f16f0e7c6d488c81852c24/google/cloud/aiplatform/gapic_version.py#L16

Reply:
Yes it should